Home page with assets

mamadOuologuem opened the pull request.

bencpeters commented on src/components/MenuBar.js:
```diff
@@ -35,17 +35,36 @@ const useStyles = makeStyles({
     width: 'fit-content',
     opacity: 1,
   },
+  button: props => ({
```
Let's destructure `props` here.

bencpeters commented on src/components/home/MapSection.js:
```diff
@@ -3,23 +3,26 @@ import { Grid, Typography } from '@material-ui/core';
+  image: {
+    position: 'relative',
+    width: 'calc(100vw - 8px)',
+    left: 'calc(-1 * (100vw - 100%) / 2)', // to centered the div
```
Didn't we change this in the last PR?

mamadOuologuem replied:
No it is not the same this is for the map image (yes it is a copy paste)

mamadOuologuem commented:
> Mostly looks good.
> 
> I'm not sure we want to have these larger PNG files as major banner images on the homepage though. What does the size look like if we have the banner and map files as fairly high res JPEGs? Was there a reason you switched to PNGs?

Indeed they has large sizes. I changed because in JPEG the images was a bit blurred (but I will try the put them back to JPEG and see how it looks like)

mamadOuologuem commented:
> > Mostly looks good.
> > I'm not sure we want to have these larger PNG files as major banner images on the homepage though. What does the size look like if we have the banner and map files as fairly high res JPEGs? Was there a reason you switched to PNGs?
> 
> Indeed they has large sizes. I changed because in JPEG the images was a bit blurred (but I will try the put them back to JPEG and see how it looks like)

it seems okay for the banner but the map's color is changing a bit when I switched to jpg

**JPG**
![1](https://user-images.githubusercontent.com/40799514/62839982-f758c280-bc47-11e9-8fa1-bdc2df62fb15.png)
VERSUS **PNG**
![2](https://user-images.githubusercontent.com/40799514/62839980-f0ca4b00-bc47-11e9-881d-3ee97d91ea13.png)

bencpeters commented:
What program are you using to make the JPEGs? And what compression settings? You can probably play with the color profile a bit to figure out why it's changing...

What is the file size of the JPEG versions of the images?
